autogen/agentchat/groupchat.py

```diff
@@ -100,6 +101,20 @@ def select_speaker(self, last_speaker: Agent, selector: ConversableAgent):
         if not final:
             # i = self._random.randint(0, len(self._agent_names) - 1)  # randomly pick an id
             return self.next_agent(last_speaker, agents)
+
+        # Find mentions of any agents
+        mentions = dict()
+        for agent in self.agents:
+            regex = r"\b" + re.escape(agent.name) + r"\b"  # Finds agent mentions, taking word boundaries into account
+            count = len(re.findall(regex, name))
+            if count > 0:
+                mentions[agent.name] = count
```

Comment:
```suggestion
                mentions[agent.name] = count
                if len(mentions) > 1:
                    break
```
early stop to be efficient

Reply:
I refactored this logic to a separate function. This comment is no longer applicable.